Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -233,7 +233,7 @@ export default class LinearTimelineComponent extends Vue {
                         class="text-center pt-2 noTimelineEntriesText"
                         data-testid="noTimelineEntriesText"
                     >
-                        No Timeline Entries
+                        No records found whitin the selected filters.
```

Comment:
"whitin" --> "within"

Reply:
@tiagograf-AOT Can you confirm that this only shows when filters are applied? What happens when there are no filters?

Reply:
@ninosamson should it say "No records found" when there's no filter?

Reply:
if you can do that @tiagograf-AOT that would be great. 
"No records found" 
"No records found with selected filters"

Reply:
done!